src/App.js

```diff
@@ -5,41 +5,51 @@ import * as tf from "@tensorflow/tfjs";
 import * as fld from "@tensorflow-models/face-landmarks-detection";
 import { Search } from "./Search";
 
+const USER_MEDIA_CONSTRAINTS = {
+  audio: false,
+  video: {
+    facingMode: "user",
+  },
+};
+
+const DETECTOR_CONFIG = {
+  runtime: "mediapipe", // or 'tfjs'
+  solutionPath: "https://cdn.jsdelivr.net/npm/@mediapipe/face_mesh",
+  refineLandmarks: true,
+};
+
 function App() {
   const [eyesClosed, setEyesClosed] = useState(false);
   const videoRef = useRef(null);
   const detectorRef = useRef(null);
 
   const initTensorFlow = async () => {
     const video = videoRef.current;
-    const stream = await navigator.mediaDevices.getUserMedia({
-      audio: false,
-      video: {
-        facingMode: "user",
-      },
-    });
+    const stream = await navigator.mediaDevices.getUserMedia(
+      USER_MEDIA_CONSTRAINTS
+    );
     video.srcObject = stream;
-    const detectorConfig = {
-      runtime: "mediapipe", // or 'tfjs'
-      solutionPath: "https://cdn.jsdelivr.net/npm/@mediapipe/face_mesh",
-    };
+
     detectorRef.current = await fld.createDetector(
       fld.SupportedModels.MediaPipeFaceMesh,
-      detectorConfig
+      DETECTOR_CONFIG
     );
   };
+
   const animate = async () => {
-    const { closed } = await detectEyeClosure(
+    const closed = await detectEyeClosure(
```

Comment:
I was actually returning an object from this function because I was testing some stuff on mobile and wanted more insight on how much of this function was working. Considering I'm going to try one more time to get chrome iOS to work, can we leave it as is for now? I'll push the fix as soon as I figure it out or give up

Reply:
sure, updated the PR back to return `{ closed }`